etna/collections/models.py

```diff
@@ -260,8 +260,8 @@ def record(self):
         skip this record on the results page.
         """
         try:
-            return Record.search.get(iaid=self.record_iaid)
-        except (KongAPIError, SearchManagerException):
+            return Record.api.fetch(iaid=self.record_iaid)
+        except (KongAPIError, APIManagerException):
             return None
```

Comment:
We are eating the exception here:   except (KongAPIError, APIManagerException):             
                                                         return None
Maybe logit somewhere before return